doc/api/tls.md

```diff
@@ -827,16 +827,27 @@ changes:
     pr-url: https://github.com/nodejs/node/pull/26625
     description: Return the minimum cipher version, instead of a fixed string
       (`'TLSv1/SSLv3'`).
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/30637
+    description: Return the IETF cipher name as `standard_name`.
 -->
 
 * Returns: {Object}
-  * `name` {string} The name of the cipher suite.
+  * `name` {string} OpenSSL name for the cipher suite.
+  * `standard_name` {string} IETF name for the cipher suite.
   * `version` {string} The minimum TLS protocol version supported by this cipher
     suite.
 
 Returns an object containing information on the negotiated cipher suite.
 
-For example: `{ name: 'AES256-SHA', version: 'TLSv1.2' }`.
+For example:
+```json
+{
+    name: 'AES128-SHA256',
```

Comment:
This currently isn't JSON, so the `json` code block might be an issue.

Reply:
fixed